src/style.css

```diff
@@ -1 +1,77 @@
-/* Styles go here */
+html {
+  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+  --item-color: #00acdc;
+  --nav-size: 60px;
+}
+
+body {
+  display: block;
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  margin: 0;
+  background-color: #fff;
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  text-transform: uppercase;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  position: relative;
+  list-style: none;
+  margin-left: 20px;
+  min-width: fit-content;
+}
+
+#nav__first__item {
```

Comment:
you don't need to add id for reset margin-rigth, please delete it and use pseudo-class(for examle: not(), last-child, last-of-type) for .nav__item